components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -2000,4 +2008,189 @@ public static SAMLSSOServiceProviderDO getSPConfig(String tenantDomain, String i
             PrivilegedCarbonContext.endTenantFlow();
         }
     }
+
+    /**
+     * Build SAML logout request.
+     *
+     * @param serviceProviderDO SP for which the logout request is built.
+     * @param subject           Subject identifier.
+     * @param sessionId         Session index.
+     * @return Logout Request.
+     * @throws IdentityException If tenant domain is invalid.
+     */
+    public static LogoutRequest buildLogoutRequest(SAMLSSOServiceProviderDO serviceProviderDO, String subject,
+                                                   String sessionId) throws IdentityException {
+
+        String destination;
+        if (StringUtils.isNotBlank(serviceProviderDO.getSloRequestURL())) {
+            destination = serviceProviderDO.getSloRequestURL();
+            if (log.isDebugEnabled()) {
+                log.debug("Destination of the logout request is set to the " +
+                        "SLO request URL of the SP: " + serviceProviderDO.getSloRequestURL());
```

Comment:
This should print the SP name, along with logout request URL right?

Reply:
Fixed with 3bbc008dd576ca0169fde3b49fe4f505a99200ea